backend/app/routers/users.py

```diff
@@ -62,7 +62,7 @@ async def create_or_change_user_api_key(
                 status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
                 detail="Error while creating user profile and API key",
             )
-    elif not user_obj.api_key:
+    else:
         try:
             user_obj.api_key = RailUser.generate_api_key()
             await RailUser.replace(user_obj)
```

Comment:
I'm trying to minimize the use of `replace` function that updates all fields of the model (or rather replaces the object with another one), and rather use `set` function that explicitly changes only specified fields:

```
await user_obj.set({
    RailUser.api_key: RailUser.generate_api_key()
})
```

Reply:
Speaking of `RailUser`, I wonder whether we want to also add `created_at` and `updated_at` fields to this object or we don't want to bother with that for now

Reply:
This is actually not related to this PR, but Martin spotted a bug here and hijacked my PR to fix this 😆 . I am ok with changing `replace` to `set`, but I wouldn't modify any further logic for now.